Compliance workflow fails due to new pylint version which checks for W1514

The CI pipeline fails on Compliance workflow, because the new pylint version has additional checks which fail. New version pylint 2.10 checks for W1514 which is defined in [PEP-0597](https://www.python.org/dev/peps/pep-0597/). pylint is used to check the integration tests. Affected version used is `pylint-2.10.2-py3-none-any.whl`.

```
integration/conftest.py:75:34: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
integration/conftest.py:89:34: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
integration/conftest.py:104:34: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)

-----------------------------------
Your code has been rated at 9.87/10
```

--- a/tests/integration/conftest.py
+++ b/tests/integration/conftest.py
@@ -74,7 +74,8 @@ class Lwm2mServer(HelperBase):
         # pylint: disable=consider-using-with
         self.pexpectobj.logfile = open(base_path +
                                        "/lwm2mserver_log.txt",
-                                       "w")
+                                       "w",
+                                       encoding="utf-8")
 
 class Lwm2mClient(HelperBase):
     """Client subclass of HelperBase"""
@@ -88,7 +89,8 @@ class Lwm2mClient(HelperBase):
         # pylint: disable=consider-using-with
         self.pexpectobj.logfile = open(base_path +
                                        "/lwm2mclient_log.txt",
-                                       "w")
+                                       "w",
+                                       encoding="utf-8")
 
 class Lwm2mBootstrapServer(HelperBase):
     """Bootstrap-server subclass of HelperBase"""
@@ -103,7 +105,8 @@ class Lwm2mBootstrapServer(HelperBase):
         # pylint: disable=consider-using-with
         self.pexpectobj.logfile = open(base_path +
                                        "/lwm2mbootstrapserver_log.txt",
-                                       "w")
+                                       "w",
+                                       encoding="utf-8")
 
 
 @pytest.fixture

